.github/workflows/connector-node-integration.yml

```diff
@@ -0,0 +1,126 @@
+name: Connector Node Integration tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+          cache: 'maven'
+      - name: run integration tests
+        run: |
+          cd connector_node
+          mvn --batch-mode --update-snapshots clean package  -DskipTests=true
```

Comment:
Do we plan to migrate to BuildKite or still maintain this GitHub action workflow? I am ok with using GitHub action for integration tests of sinks.  cc @huangjw806

Reply:
it is a temp solution. migrate ci later